modules/react/modal/stories/stories.tsx

```diff
@@ -39,50 +39,58 @@ export const Default = () => {
-      <Modal data-testid="TestModal" heading={'Delete Item'} {...modalProps}>
+      <Modal heading={'Delete Item'} {...modalProps}>
         <p>Are you sure you'd like to delete the item titled 'My Item'?</p>
-        <SecondaryButton {...popup1.targetProps}>Open Popup to reach Delete button</SecondaryButton>
-        <SecondaryButton {...popup2.targetProps}>Non-hidable Popup</SecondaryButton>
+        <Popup.Target model={popup1}>Open Popup to reach Delete button</Popup.Target>
+        <Popup.Target model={popup2}>Non-hidable Popup</Popup.Target>
         <Tooltip title={'Not so sure'} type={'muted'}>
           <SecondaryButton onClick={closeModal}>Cancel</SecondaryButton>
         </Tooltip>
       </Modal>
-      <Popper {...popup1.popperProps}>
-        <Popup heading="Really Delete" handleClose={popup1.closePopup}>
-          Pressing 'Delete' will close the modal
-          <Tooltip
-            placement="left"
-            title={'Really, Really, Really, Really, Really sure'}
-            type={'muted'}
-          >
-            <DeleteButton style={{marginRight: '16px'}} onClick={closeModal}>
-              Delete
-            </DeleteButton>
-          </Tooltip>
-        </Popup>
-      </Popper>
-      <Popper {...popup2.popperProps}>
-        <Popup heading="Does Not Hide On Click Outside" handleClose={popup2.closePopup}>
-          Pressing 'Delete' will close the modal
-          <Tooltip
-            placement="left"
-            title={'Really, Really, Really, Really, Really sure'}
-            type={'muted'}
-          >
-            <DeleteButton style={{marginRight: '16px'}} onClick={closeModal}>
-              Delete
-            </DeleteButton>
-          </Tooltip>
-        </Popup>
-      </Popper>
+      <Popup.Popper model={popup1}>
+        <Popup.Card model={popup1}>
+          <Popup.CloseIcon model={popup1} aria-label="Close" />
+          <Popup.Heading model={popup1}>Really Delete</Popup.Heading>
+          <Popup.Body>
+            Pressing 'Delete' will close the modal
+            <Tooltip
+              placement="left"
+              title={'Really, Really, Really, Really, Really sure'}
+              type={'muted'}
+            >
+              <DeleteButton style={{marginRight: '16px'}} onClick={closeModal}>
+                Delete
+              </DeleteButton>
+            </Tooltip>
+          </Popup.Body>
+        </Popup.Card>
+      </Popup.Popper>
+      <Popup.Popper model={popup2}>
+        <Popup.Card model={popup2}>
+          <Popup.CloseIcon model={popup2} aria-label="Close" />
+          <Popup.Heading model={popup2}>Does Not Hide On Click Outside</Popup.Heading>
+          <Popup.Body>
+            Pressing 'Delete' will close the modal
+            <Tooltip
+              placement="left"
+              title={'Really, Really, Really, Really, Really sure'}
+              type={'muted'}
+            >
+              <DeleteButton style={{marginRight: '16px'}} onClick={closeModal}>
```

Comment:
It's odd that clicking this doesn't close itself as well.

Reply:
I fixed this and moved to a testing story